about.html

```diff
@@ -1,115 +1,254 @@
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@400;800;900&family=Lato:wght@400;700;900&display=swap"
+      rel="stylesheet"
+    />
+    <!-- font awesome CDN -->
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.0.7/css/all.css"
+    />
+    <!-- local css files -->
+    <link rel="stylesheet" href="./css/desktop.css" />
+    <link rel="stylesheet" href="./css/styles.css" />
     <title>About the Conference</title>
-</head>
-<body>
+  </head>
+  <body>
     <header>
-        <div class="w-100">
-            <!-- first Nav -->
-            <nav class="navbar d-none d-sm-none d-md-none d-lg-block bg-dark py-1 text-light">
-                <div class="container d-flex justify-content-end col-lg-9">
-                    <ul class="list-unstyled d-flex col-lg-6 justify-content-between fs-6">
-                        <li>
-                            <a href="#" class="nav-link text-reset"><i class="fab fa-twitter-square "></i></a>
-                        </li>
-                        <li>
-                            <a href="#" class="nav-link text-reset"><i class="fab fa-facebook-square"></i></a>
-                        </li>
-                        <li>
-                            <a href="#" class="nav-link text-reset">English</a>
-                        </li>
-                        <li>
-                            <a href="#" class="nav-link text-reset">My Page</a>
-                        </li>
-                        <li>
-                            <a href="#" class="nav-link text-reset">Logout</a>
-                        </li>
-                    </ul>
-                </div>
-                
-            </nav>
+      <div class="w-100">
+        <!-- first Nav -->
+        <nav
+          class="navbar d-none d-sm-none d-md-none d-lg-block bg-dark py-1 text-light"
+        >
+          <div class="container d-flex justify-content-end col-lg-9">
+            <ul
+              id="first-nav"
+              class="list-unstyled d-flex col-lg-6 justify-content-between fs-6"
+            >
+              <li>
+                <a href="#" class="nav-link text-reset"
+                  ><i class="fab fa-twitter-square"></i
+                ></a>
+              </li>
+              <li>
+                <a href="#" class="nav-link text-reset"
+                  ><i class="fab fa-facebook-square"></i
+                ></a>
+              </li>
+              <li>
+                <a href="#" class="nav-link text-reset">English</a>
+              </li>
+              <li>
+                <a href="#" class="nav-link text-reset">My Page</a>
+              </li>
+              <li>
+                <a href="#" class="nav-link text-reset">Logout</a>
+              </li>
+            </ul>
+          </div>
+        </nav>
+      </div>
```

Comment:
- [ ] You are almost there! Let's decrease the height of the black navigation bar, as the first reviewer said it's still looking fairly different from the template:

Template:
![Captura de pantalla 2022-05-20 a la(s) 1 12 30 p  m](https://user-images.githubusercontent.com/83717285/169588267-0488c1de-b2be-45ea-a1d5-ffe5b12933c0.png)

Your design:
![Captura de pantalla 2022-05-20 a la(s) 1 07 03 p  m](https://user-images.githubusercontent.com/83717285/169588294-b2c017d2-4930-47ae-8f4a-4a6bc95eee53.png)